boards/arm/sam_v71_xult/doc/index.rst

```diff
@@ -0,0 +1,181 @@
+- One reset and two user pushbutton
+- Two yellow user LED
+
+Supported Features
+==================
+
+The sam_v71_xplained_ultra board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | ADC via AFEC                        |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig
+:zephyr_file:`boards/arm/sam_v71_xult/sam_v71_xult_defconfig`.
+
+Connections and IOs
+===================
+
+The `SAMV71-XULT User Guide`_ has detailed information about board
+connections.
+
+System Clock
+============
+
+The SAM V71 MCU is configured to use the 12 MHz external oscillator on the
+board with the on-chip PLL to generate a 300 MHz system clock.
+
+Serial Port
+===========
+
+The ATSAMV71Q21 MCU has five UARTs and three USARTs. One of the USARTs is
+configured for the console and is available as a Virtual COM Port via EDBG USB
+chip.
+
+Programming and Debugging
+*************************
+
+Flashing the Zephyr project onto SAM V71 MCU requires the `OpenOCD tool`_.
+Support for Atmel SAM E microcontroller series was added in OpenOCD release
+0.10.0, which was added in Zephyr SDK 0.9.2.
+
+By default a factory new SAM V71 chip will boot SAM-BA boot loader located in
```

Comment:
Let's add a link to the document since you have defined it at the bottom of the page:
```
 ... will boot the `SAM-BA`_ boot loader ...
```

Reply:
Thank you for all your support :D